src/update.ts

```diff
@@ -372,9 +360,7 @@ export const makeUpdate =
               return new CollectionToken([
                 new StringToken(wrapQuotes(column.getSnakeCaseName())),
                 new StringToken(`=`),
-                value && typeof value === `object` && 'toTokens' in value
-                  ? value.toTokens()
-                  : new ParameterToken(value),
+                ...(isTokenable(value) ? value.toTokens() : [new ParameterToken(value)]),
```

Comment:
This was a bug I discovered as part of testing.  `CollectionToken` expects an array of tokens in the constructor.  If the condition here is true, then this would end up as an array of tokens rather than a single token.  The solution here was to make this condition always evaluate to an array of tokens, and spread it into the array for the collection token.